setup.py

```diff
@@ -1,6 +1,19 @@
 from setuptools import setup, find_packages
 
 required_packages = ["graphql-core>=2.1", "promise"]
+tests_require = [
+    "pytest",
+    "pytest-benchmark",
+    "pytest-cov",
+    "pytest-mock",
+    "snapshottest",
```

Comment:
are all of these really required to run tests? In the before diff there was only pytest.

Reply:
Probably not all of them, but we weren't running coverage before. (coverage is at 66pct with the current config, which seems low for a graphql python lib)